src/components/FilterPanel.jsx

```diff
@@ -70,12 +100,36 @@ export default function FilterPanel({ onClose }) {
             expandIcon={<ExpandMoreIcon />}
             aria-controls={name + '-content'}
             id={name + '-header'}>
-            <Typography>{name}</Typography>
+            <Grid container alignItems="center" justifyContent="space-between">
```

Comment:
Tried using `Stack` and it was not styling it how I wanted it despite using the same properties